repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
Last I checked the apiserver upgrades the http connection to SPDY, for
things like 'exec' (possibly elsewhere) for better bidirectional stream
support.

On Mon, Oct 5, 2015 at 4:38 PM, Karl Isenberg <notifications@github.com>
wrote:

> In repo/packages/K/kubernetes/1/config.json
> <https://github.com/mesosphere/multiverse/pull/13#discussion_r41194374>:
>
> > @@ -81,6 +81,15 @@
> >            "minimum": 0,
> >            "type": "integer"
> >          },
> > +        "resource-roles": {
>
> @jdef <https://github.com/jdef> mentioned needing SPDY support. I haven't
> verified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/13/files#r41194374>.
>